### PR TITLE
Update duckduckgo.po es_MX

### PR DESCRIPTION
**Pull request by LuisAlfredo92**
I used Online PO file editor

### CODE REVIEW

File: locales/es_MX/LC_MESSAGES/duckduckgo.po [hunk 1]
```diff
@@ -73,57 +71,63 @@ msgstr "%snos"
 #. Part of the sentence: "Use <b>us.</b>"
 msgctxt "useus"
 msgid "%s us."
-msgstr "%s nosotros."
+msgstr "%snos."
```
Comment (GioSensation) [inline locales/es_MX/LC_MESSAGES/duckduckgo.po:74]: A space is missing.

```suggestion
msgstr "%s nos."
```

File: locales/es_MX/LC_MESSAGES/duckduckgo.po [hunk 2]
```diff
@@ -321,34 +328,34 @@ msgstr "Ubicación anónima habilitada."
...
 
-#. In feedback page, first box. https://duckduckgo.com/feedback 
+#. In feedback page, first box. https://duckduckgo.com/feedback
 msgid "Answers to most DuckDuckGo questions, like:"
 msgstr "Respuestas a la mayoría de las preguntas sobre DuckDuckGo, como:"
 
 msgctxt "video-duration"
 msgid "Any Duration"
-msgstr "Cualquier Duración"
+msgstr "Cualquier duración"
```
Comment (GioSensation) [inline locales/es_MX/LC_MESSAGES/duckduckgo.po:337]: It's better to keep the capitalization of the original string.

```suggestion
msgstr "Cualquier Duración"
```

File: locales/es_MX/LC_MESSAGES/duckduckgo.po [hunk 3]
```diff
@@ -321,34 +328,34 @@ msgstr "Ubicación anónima habilitada."
...
 
 msgctxt "video-license"
 msgid "Any License"
-msgstr "Cualquier Licencia"
+msgstr "Cualquier licencia"
```
Comment (GioSensation) [inline locales/es_MX/LC_MESSAGES/duckduckgo.po:341]:
```suggestion
msgstr "Cualquier Licencia"
```

File: locales/es_MX/LC_MESSAGES/duckduckgo.po [hunk 4]
```diff
@@ -321,34 +328,34 @@ msgstr "Ubicación anónima habilitada."
...
 
 msgctxt "video-resolution"
 msgid "Any Resolution"
-msgstr "Cualquier Resolución"
+msgstr "Cualquier resolución"
```
Comment (GioSensation) [inline locales/es_MX/LC_MESSAGES/duckduckgo.po:345]:
```suggestion
msgstr "Cualquier Resolución"
```

File: locales/es_MX/LC_MESSAGES/duckduckgo.po [hunk 5]
```diff
@@ -321,34 +328,34 @@ msgstr "Ubicación anónima habilitada."
...
 
 #. Option for the filter-by-date search.
 msgid "Any Time"
 msgstr "En cualquier momento"
 
 msgid "Anywhere"
-msgstr "Donde Sea"
+msgstr "Donde sea"
```
Comment (GioSensation) [inline locales/es_MX/LC_MESSAGES/duckduckgo.po:352]:
```suggestion
msgstr "Donde Sea"
```

File: locales/es_MX/LC_MESSAGES/duckduckgo.po [hunk 6]
```diff
@@ -321,34 +328,34 @@ msgstr "Ubicación anónima habilitada."
...
 
 msgid "App"
 msgstr "Aplicación"
 
 msgid "App and Extension"
-msgstr ""
+msgstr "Aplicación y extensión"
```
Comment (GioSensation) [inline locales/es_MX/LC_MESSAGES/duckduckgo.po:358]:
```suggestion
msgstr "Aplicación y Extensión"
```

File: locales/es_MX/LC_MESSAGES/duckduckgo.po [hunk 7]
```diff
@@ -399,19 +409,19 @@ msgstr "Sugerencia automática"
 #. http://i.imgur.com/OZIO1I9.png
 msgctxt "settings"
 msgid "Automatically open relevant Instant Answers"
-msgstr "Abrir automáticamente Respuestas Instantáneas relevantes"
+msgstr "Abrir automáticamente respuestas instantáneas relevantes"
```
Comment (GioSensation) [inline locales/es_MX/LC_MESSAGES/duckduckgo.po:412]:
```suggestion
msgstr "Abrir automáticamente Respuestas Instantáneas relevantes"
```

File: locales/es_MX/LC_MESSAGES/duckduckgo.po [hunk 8]
```diff
@@ -614,7 +631,7 @@ msgid "Canada"
 msgstr "Canadá"
 
 msgid "Canada (fr)"
-msgstr ""
+msgstr "Canadá (Fr)"
```
Comment (GioSensation) [inline locales/es_MX/LC_MESSAGES/duckduckgo.po:634]: I understand your point in having the language codes capitalized, but as I said, it's better to keep things consistent with the original strings.

```suggestion
msgstr "Canadá (fr)"
```

File: locales/es_MX/LC_MESSAGES/duckduckgo.po [hunk 9]
```diff
@@ -562,18 +573,24 @@ msgid "Brown"
...
-msgstr ""
+msgstr "Navega como siempre"
 
 msgid ""
 "Browse as usual, and we'll take care of the rest. We bundled our search "
 "engine, tracker blocker, and encryption enforcer into one %s%s Extension%s."
 msgstr ""
+"Navega como siempre, nosotros nos encargamos del resto. Incluimos nuestro "
+"motor de búsqueda, bloqueador de rastreadores y forzador de cifrado en una "
+"sola %s%s Extensión%s%s."
```
Comment (GioSensation) [inline locales/es_MX/LC_MESSAGES/duckduckgo.po:584]:
```suggestion
"sola %s%s Extensión%s."
```

File: locales/es_MX/LC_MESSAGES/duckduckgo.po [hunk 10]
```diff
@@ -3668,29 +3708,29 @@ msgstr "Selecciona %sPreferencias%s."
 
 msgctxt "mobile homepage banner"
 msgid "Select %sSearch Engine%s"
-msgstr ""
+msgstr "Selecciona %sMotor de búsqueda%s"
 
 msgctxt "mobile homepage banner"
 msgid "Select %sSearch engine%s, then %sOthers...%s"
-msgstr ""
+msgstr "Selecciona %sMotor de búsqueda%s, luego $sOtros...%s"
```
Comment (GioSensation) [inline locales/es_MX/LC_MESSAGES/duckduckgo.po:3715]:
```suggestion
msgstr "Selecciona %sMotor de búsqueda%s, luego %sOtros...%s"
```

File: locales/es_MX/LC_MESSAGES/duckduckgo.po [hunk 11]
```diff
@@ -3668,29 +3708,29 @@ msgstr "Selecciona %sPreferencias%s."
...
-msgstr ""
+msgstr "Selecciona %sAjustes%s"
 
 msgid "Select %sSettings%s from the drop-down menu."
 msgstr "Seleccione %sOpciones%s del menú desplegable."
 
 msgctxt "mobile homepage banner"
 msgid "Select %sSettings%s, then %sAdvanced settings%s"
-msgstr ""
+msgstr "Selecciona %sAjustes%s, luego $sOpciones avanzadas%s"
```
Comment (GioSensation) [inline locales/es_MX/LC_MESSAGES/duckduckgo.po:3729]:
```suggestion
msgstr "Selecciona %sAjustes%s, luego %sOpciones avanzadas%s"
```

File: locales/es_MX/LC_MESSAGES/duckduckgo.po [hunk 12]
```diff
@@ -3668,29 +3708,29 @@ msgstr "Selecciona %sPreferencias%s."
...
 
 msgctxt "mobile homepage banner"
 msgid "Select %sSettings%s, then %sDefault search engine%s"
-msgstr ""
+msgstr "Selecciona %sAjustes%s, luego $sMotor de búsqueda por defecto%s"
```
Comment (GioSensation) [inline locales/es_MX/LC_MESSAGES/duckduckgo.po:3733]:
```suggestion
msgstr "Selecciona %sAjustes%s, luego %sMotor de búsqueda por defecto%s"
```

File: locales/es_MX/LC_MESSAGES/duckduckgo.po [hunk 13]
```diff
@@ -2431,7 +2460,7 @@ msgstr "Más información"
 
 msgctxt "Ads GDPR, internal use only. Do not change"
 msgid "More Info"
-msgstr ""
+msgstr "Más información"
```
Comment (GioSensation) [inline locales/es_MX/LC_MESSAGES/duckduckgo.po:2463]: This string is for internal use only and must not be changed.

```suggestion
msgstr ""
```

File: locales/es_MX/LC_MESSAGES/duckduckgo.po [hunk 14]
```diff
@@ -4562,6 +4619,8 @@ msgid ""
 "Viewing ads is privacy protected by DuckDuckGo. Ad clicks are managed by "
 "Microsoft's ad network"
 msgstr ""
+"Viewing ads is privacy protected by DuckDuckGo. Ad clicks are managed by "
+"Microsoft's ad network"
```
Comment (GioSensation) [inline locales/es_MX/LC_MESSAGES/duckduckgo.po:4623]: This string is for internal use only and must be left empty.

```suggestion
```

File: locales/es_MX/LC_MESSAGES/duckduckgo.po [hunk 15]
```diff
@@ -73,57 +71,63 @@ msgstr "%snos"
...
 "%sClick here%s to try again, if you think there should be results for this "
 "search."
 msgstr ""
+"%sClica aquí%s para intentarlo de nuevo, si crees que debería haber "
+"resultados de esta búsqueda."
 
 msgctxt "frontpage"
 msgid "%sHelp Spread DuckDuckGo!%s"
-msgstr ""
+msgstr "%s¡Ayuda a difundir DuckDUckGo!%s"
```
Comment (GioSensation) [inline locales/es_MX/LC_MESSAGES/duckduckgo.po:109]:
```suggestion
msgstr "%s¡Ayuda a difundir DuckDuckGo!%s"
```

**Comment by GioSensation** [comment]
Thanks for your contribution, @LuisAlfredo92. Much appreciated. I have fixed the most problematic things so that the CI checks are now passing. There are still a lot of capitalization inconsistencies. It's usually better to keep the same capitalization of the original string. If you use a PO editor such as [Poedit](https://poedit.net), it will highlight such inconsistencies for you. I ask you to please go through the file again and try to maintain the original capitalization. Then assign this back to me and I will merge it. Thanks again for your help with this. 🙇

**Comment by GioSensation** [comment]
Hey @LuisAlfredo92. Thanks for looking into this again. Much appreciated. I noticed that you accidentally committed the [.mo file](https://github.com/duckduckgo/duckduckgo-locales/pull/351/files#diff-ab077ba71e184117f0369e2685e89ca6). Please remove that, and I will merge the PR.

**Comment by LuisAlfredo92** [comment]
Everything is ready, thanks for all the support

**Comment by GioSensation** [comment]
Thanks! Merged. 🎉

**Pull request merged**